src/main/scala/beam/utils/scenario/urbansim/CsvScenarioReader.scala

```diff
@@ -61,7 +61,7 @@ object CsvScenarioReader extends UrbanSimScenarioReader with LazyLogging {
 
   private def toPlanInfo(rec: java.util.Map[String, String]): PlanElement = {
     // Somehow Plan file has columns in camelCase, not snake_case
-    val tripId = Option(rec.get("tripId"))
+    val tripId = Option(rec.get("tripId").filter(x => (x.isDigit || x.equals('.'))))
```

Comment:
What does dot means? Is it some special case for trip ID? I do not see any special logic to process dots as IDs

Reply:
Since our input is float number, it will have dot. If we filter the dot out, it will be wrong